### DISCUSSION TITLE
Loading Image | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#946] [Q&A]
# Loading Image | Seeed Studio Wiki

This repo describes how to load and display images from the SD card to the TFT LCD screen on Wio Terminal. This can be very useful implementation to your design and can be used in various places!

https://wiki.seeedstudio.com/Wio-Terminal-LCD-Loading-Image/

**Comment by FyrbyAdditive** [comment]
The python script does not work on macOS. I converted the image to 16 bit 565 instead using an online converter and it just generates junk on the screen.

**Reply by FyrbyAdditive** [reply]
I needed to modify line 33 and add another line before it as the script as posted seems to not return RGB values but something else entirely.

            imrgb = im.convert("RGB") # This is the new line
            v = [cvt(r, g, b) for (r, g, b) in imrgb.getdata()] # This line has had the im variable changed to imrgb
